src/hooks/wallets/wallets.ts

```diff
@@ -38,7 +39,17 @@ const walletConnectV2 = (chain: ChainInfo) => {
   })
 }
 
+const metamaskSDKWallet = metamaskSDK({
+  options: {
+    extensionOnly: true,
+    dappMetadata: {
+      name: 'Safe',
+    },
+  },
+})
+
 const WALLET_MODULES: Partial<{ [key in WALLET_KEYS]: (chain: ChainInfo) => WalletInit }> = {
+  [WALLET_KEYS.METAMASK]: () => metamaskSDKWallet,
```

Comment:
Dis not a function call like other modules?

Reply:
It is I just extracted it because we are passing a rather large options object otherwise it would look like this:

```
[WALLET_KEYS.METAMASK]: () =>
    metamaskSDK({
      options: {
        extensionOnly: true,
        dappMetadata: {
          name: 'Safe Wallet',
        },
      },
    }),
```

Reply:
I would keep it consistent with the other modules.